app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/RadioInput/__tests__/Radioinput.test.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { fireEvent, render, screen } from '@testing-library/react';
+import React, { ChangeEvent } from 'react';
+import RadioInput from 'components/WranglerGrid/SelectColumnPanel/DataTable/RadioInput';
+
+describe('It should render ', () => {
```

Comment:
These tests are pointless, you're checking to see if the material ui library works which we absolutely don't want to do.

Reply:
Removed

Reply:
we have removed this test file as well
the component is moved to its parent where we are returning either Radio or Checkbox based on required condition

we have add the conditional rendering test case in parent component